conduce/api.py

```diff
@@ -598,6 +598,95 @@ def ingest_entities(dataset_id, entity_list, **kwargs):
     return _ingest_entity_set(dataset_id, entity_set, **kwargs)
 
 
+def insert_transaction(dataset_id, entity_set, **kwargs):
```

Comment:
It just occurred to me that these names may be too vague for the python API.  Should `insert_transaction` become `insert_dataset_transaction` and so on?

Reply:
I think `insert_transaction` is fine.  We don't use transaction in any context other than around a dataset, correct?  What else might we add that has the concept of a transaction that must be inserted/appended/posted?

Reply:
No idea, terms like transaction and backend just don't seem specific (self-documenting) enough.

Reply:
For the backend APIs I used `create_dataset_backend`, `add_elasticsearch_store`, and `process_transactions`